doc/api/n-api.md

```diff
@@ -541,6 +541,19 @@ This API returns true if an exception is pending.
 
 This API can be called even if there is a pending JavaScript exception.
 
+#### napi_fatal_exception
+<!-- YAML
+added: TBA
```

Comment:
`REPLACEME` is the magic word that gets picked up by the release tooling, please change to that :)

Reply:
nice! i was offline and trying hard to remember what it was